### ISSUE TITLE
Paths cannot be copied from the tutorials in Rev

### ISSUE THREAD

**Issue by bjoelle** [revbayes/revbayes#50]
due to path separators being different between Unix OSes and Windows

**Comment by wrightaprilm** [comment]
As a note: If you're on windows and launch via GitBash or a similar terminal emulator, *NIX style paths will work. Except the `~`. If you click the executable on Windows to start RevBayes, you should be able to used mac-style relative paths once your path is set. For example, this morning, I set my path like so:

```
setwd('c:\\Users\\april\\revbayes\\')
```

And afterwards could do:

```
source("scripts/mcmc_example.Rev")
``` 

And this behavior varies between Windows machines, as well. Some will allow NIX-style separators, some will not. It's very frustrating in my systematics class, which has 4 different Windows OSes present. Given that relative paths seem to work NIX-style once the working directory is set, perhaps pointing users to the [setup tutorial](https://revbayes.github.io/tutorials/tutorial_structure/) is the way to go?

**Comment by davidcerny** [comment]
This also seems to have been resolved; for v1.2.4 on Windows 10 Pro for Workstations (v22H2), I can not only use relative paths with Unix separators (like `source("scripts/mcmc_example.Rev")`), but also this:

```
setwd("C:/Users/Graham/Downloads/")
```

I didn't have to install any extra software, just double-click the executable. @bjoelle Would you be okay with closing this?

**Comment by bjoelle** [comment]
yes, this is resolved I think
